controlpanel/api/cluster.py

```diff
@@ -64,6 +67,13 @@
     },
 }
 
+TOOL_DEPLOYING = 'Deploying'
```

Comment:
Nice that we use constants.

There is something that feels odd about having these constants in the `api.cluster` module tho for some reason.
I wonder if a better place for them would be inside the `ToolDeployment` class, e.g. `ToolDeployment.FAILED` or `ToolDeployment.IDLED` 🤷‍♂

Reply:
Yeah, I agree, but if the constants are defined in `models/tool.py` and used in `cluster.py` you get a circular import. That's why I also defined the exceptions in `cluster`. Maybe I should move them into another module that both the model and `cluster` can import from.

Reply:
I see what you mean about the cycle now 🤒- not the end of the world anyway.